project_parent_task_filter/views/project_task.xml

```diff
@@ -20,4 +20,28 @@
             </xpath>
          </field>
      </record>
+
+    <record id="view_task_kanban_subtask" model="ir.ui.view">
+        <field name="name">project.task.kanban</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_kanban"/>
+        <field name="arch" type="xml">
+            <field name="color" position="after">
+                <field name="subtask_count"/>
+                <field name="parent_id"/>
+            </field>
+            <xpath expr="//div[hasclass('oe_kanban_bottom_left')]" position="inside">
+                <div class="pull-right" groups="base.group_user" style="margin-left: 10px;"> 
+                    <t attrs="{'invisible': [('parent_id', '!=', False)]}"
```

Comment:
Sub tasks can also have sub tasks right?

Reply:
I think subtasks can't hace subtasks... See the view form of a subtask
![image](https://user-images.githubusercontent.com/24428859/56576574-06e4ca80-65c9-11e9-8c03-07965c035a52.png)

Reply:
They can have sub tasks (you can assign a sub-task as a parent task), but they appear as sub-tasks of their parent (because of `child_of` operator). In any case, I think this approach is OK as it follows the same logic as Odoo core: don't show sub-task's sub-tasks, but rather show them as subtasks of the parent task :)